frontend/plan/src/syncutils.js

```diff
@@ -53,7 +53,7 @@ const initiateSync = async (store) => {
                 handleSemester(options.SEMESTER);
             })
             .catch(() => {
-                handleSemester("2020C");
+                handleSemester("2020A");
```

Comment:
Can we pop up a modal in this case? or something saying that something went wrong instead of defaulting to a semester that might not be right?